src/main/ui-modules/options/all.xml

```diff
@@ -18,40 +24,24 @@
       See Usage Notes of http://docs.marklogic.com/cts:word-query for details.
       Also note that forcing stemmed searches will disable wildcarded searches.
     -->
+    <!-- NOTE: You may need to enable features for DHF/DHS final database to use features like stemming and wildcards -->
     <!--
     <term-option>punctuation-insensitive</term-option>
     <term-option>stemmed</term-option>
     -->
   </term>
 
-  <!-- Default grammer -->
-  <grammar>
-    <quotation>"</quotation>
-    <implicit>
-      <cts:and-query strength="20" xmlns:cts="http://marklogic.com/cts"/>
-    </implicit>
-    <starter strength="30" apply="grouping" delimiter=")">(</starter>
-    <starter strength="40" apply="prefix" element="cts:not-query">-</starter>
-    <joiner strength="10" apply="infix" element="cts:or-query" tokenize="word">OR</joiner>
-    <joiner strength="20" apply="infix" element="cts:and-query" tokenize="word">AND</joiner>
-    <joiner strength="30" apply="infix" element="cts:near-query" tokenize="word">NEAR</joiner>
-    <joiner strength="30" apply="near2" consume="2" element="cts:near-query">NEAR/</joiner>
-    <joiner strength="50" apply="constraint">:</joiner>
-    <joiner strength="50" apply="constraint" compare="LT" tokenize="word">LT</joiner>
-    <joiner strength="50" apply="constraint" compare="LE" tokenize="word">LE</joiner>
-    <joiner strength="50" apply="constraint" compare="GT" tokenize="word">GT</joiner>
-    <joiner strength="50" apply="constraint" compare="GE" tokenize="word">GE</joiner>
-    <joiner strength="50" apply="constraint" compare="NE" tokenize="word">NE</joiner>
-  </grammar>
-
   <!-- Default sort order. You can also associate states with sort orders to allow switching sort orders -->
   <sort-order direction="descending">
     <score/>
   </sort-order>
 
   <!-- Not used, but the default snippeting behavior if no state (see compact/detailed below) was specified -->
   <transform-results apply="snippet">
-    <preferred-elements><element ns="" name="body"/></preferred-elements>
+    <preferred-matches>
+      <element ns="http://marklogic.com/entity-services" name="instance"/>
+      <json-property>instance</json-property>
```

Comment:
It is a separate ticket to look closer at this, but we need to drop the json-property bit at the least. It causes search to fail, which is worse than just showing snippets starting with header data..

Reply:
Delete       <json-property>instance</json-property>

Reply:
Deleted line.